urllib3/gae_connectionpool.py

```
@@ -0,0 +1,96 @@
+from .connectionpool import (
+    ConnectionPool,
+    _Default,
+    get_host)
+
+from .exceptions import (
+    ProtocolError,
+    HostChangedError,
+    LocationValueError,
+    MaxRetryError,
+    SSLError,
+    TimeoutError,
+    InsecureRequestWarning,
+)
+
+from .packages.six import BytesIO
+from .request import RequestMethods
+from .response import HTTPResponse
+from .util.timeout import Timeout
+from .util.url import Url
+
+try:
+    from google.appengine.api import urlfetch
+except ImportError:
+    urlfetch = None
+
+
+class GAEConnectionPool(ConnectionPool, RequestMethods):
+
+    def __init__(self, host, port=None, scheme='http', timeout=5, headers=None,
+                 retries=None, validate_certificate=False):
+
+        if not urlfetch:
+            raise EnvironmentError(
+                "URLFetch is not available in this environment.")
+
+        ConnectionPool.__init__(self, host, port)
+        RequestMethods.__init__(self, headers)
+
+        # URLFetch doesn't support fine-grained control over timeouts.
+        if isinstance(timeout, Timeout):
+            timeout = timeout.total
```

Agreed w/ not silently ignoring, but URLFetch's default of 5s is really low and a lot of users will want to change this. What's the recommended way to do this?